examples/custom_client.rs

```diff
@@ -0,0 +1,43 @@
+//! This example demonstrates how to use the same connection pool for both obtaining the tokens and
+//! using the API that these tokens authorize. In most cases e.g. obtaining the service account key
+//! will already establish a keep-alive http connection, so the succeeding API call should be much
+//! faster.
+//!
+//! It is also a better use of resources (memory, sockets, etc.)
+
+async fn r#use<C>(
```

Comment:
is `r#use` some new syntax I'm not aware of, or did you choose this name intentionally? It reminds me of the raw string literal syntax

Reply:
It is a raw identifier, as `use` is otherwise a reserved keyword. I had a hard time coming up with another name here. Open to suggestions.